R/PackageData2Csv.R

```diff
@@ -0,0 +1,89 @@
+#' Create a CSV variant of .rda files
+#'
+#' The function should be run with a path to a package directory.
+#' It will then look through the `data` directory of the package,
+#' and for all data sets that are data frames, create CSV variants
+#' in a `data-csv` directory.
+#'
+#' @param pkg The R package where we'd like to generate CSVs of any
+#'   data frames.
+#' @param dir A character string representing the path to the folder.
+#'   where the CSV files should be written. If no such directory
+#'   exists, one will be created (recursively).
+#' @param overwrite Boolean to indicate if to overwrite any existing
+#'   files that have conflicting names in the directory specified.
+#' @export
+#' @examples
+#'
+#' \dontrun{
+#'   PackageData2Csv("openintro")
+#'   list.files("data-csv")
+#' }
+PackageData2Csv <- function(pkg, dir = "data-csv", overwrite = FALSE) {
+  stopifnot(pkg %in% dimnames(utils::installed.packages())[[1]])
+  stopifnot(dir != "")
+  data_sets <- utils::data(package = "openintro")$results[, 3]
+  if ("zzq" %in% data_sets) {
```

Comment:
Can you use a more informative text string than `zzq` so it's obvious what it's for?

Reply:
Sure, I'm updating the name to `tmp_data`.